src/upload/admin/view/template/extension/payment/webpay_rest.twig

```diff
@@ -444,9 +444,11 @@
                             </div>
                             <div class="info-column">
                                 <div class="info-value">
-                                    {% if loghandler.config.status == true %}
+                                    {% if log_status %}
```

Comment:
Is there a way to disable it?

Reply:
This cannot be disabled, it will be removed from the frontend as it does not make any contribution.